pkgs/by-name/fi/finamp/package.nix

```
@@ -0,0 +1,73 @@
+{ lib
+, flutter
+, mpv
+, xdg-user-dirs
+, patchelf
+, fetchFromGitHub
+, copyDesktopItems
+, makeDesktopItem
+}:
+let
+  version = "0.9.8-beta";
+in
+flutter.buildFlutterApplication {
```

The flutter version should pin to upstream version. See https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/dart.section.md#flutter-applications-ssec-dart-flutter.

```suggestion
flutter322.buildFlutterApplication {
```

You can find the version in it's workflow run log (of the `0.9.8-beta` commit): https://github.com/jmshrv/finamp/actions/runs/9504242145/job/26196438043#step:5:38

---

Fixed